src/property-filter/token-editor-inputs.tsx

```diff
@@ -171,7 +171,7 @@ function ValueInputAuto({
   return (
     <InternalAutosuggest
       enteredTextLabel={i18nStrings.enteredTextLabel}
-      value={matchedOption?.label ?? value ?? ''}
+      value={matchedOption?.label ?? (value ?? '') + ''}
```

Comment:
`value` has already been sanitized in line 158, so can we pass it as is here, i.e, just `value` instead of `(value ?? '') + ''`?